04-single-hop-6lowpan-icmp/test_spec04.py

```diff
@@ -286,3 +286,88 @@ def test_task10(riot_ctrl):
     time.sleep(60)
     assert pktbuf(pinged).is_empty()
     assert pktbuf(pinger).is_empty()
+
+
+@pytest.mark.iotlab_creds
+# nodes passed to riot_ctrl fixture
+@pytest.mark.parametrize('nodes',
+                         [pytest.param(['nrf52840dk', 'iotlab-m3',
+                                        'iotlab-m3'])],
```

Comment:
This node selection does not express the nodes described in the Specs. There it asks for a "Zolertia Remote". If the availability in the IoT-Lab is the problem, this tasks should rather be `local_only`.

Reply:
I forgot to update those. I will do it quickly tomorrow morning

Reply:
done!